odoosync.php

```diff
@@ -143,12 +143,36 @@ function odoosync_civicrm_navigationMenu(&$menu) {
 /**
  * Implements hook_civicrm_postProcess().
  *
- * @throws \CiviCRM_API3_Exception
  */
 function odoosync_civicrm_postProcess($formName, &$form) {
-  $formAction = $form->getAction();
+  postProcessContactInlineForms($formName, $form);
+  postProcessContactTaskDelete($formName, $form);
+  postProcessContactForm($formName, $form);
+}
+
+/**
+ * Updates contact sync information through editing contact form inline-blocks
+ *
+ * @param $formName
+ * @param $form
+ */
+function postProcessContactInlineForms($formName, $form) {
+  $isInlineContactForm = preg_match('/^CRM_Contact_Form_Inline_/', $formName);
+  if ($isInlineContactForm && ($form->getAction() == CRM_Core_Action::UPDATE || CRM_Core_Action::ADD || CRM_Core_Action::DELETE)) {
```

Comment:
but the only operation is **update** when you are doing inline edit, correct ?

Reply:
We can not only edit the data in contact inline forms.
For example, we can add new address, update it, and then delete it (I've tested all of them, functionality is separated into three different form actions)